pydis_site/apps/api/pagination.py

```diff
@@ -0,0 +1,49 @@
+import typing
+
+from rest_framework.pagination import LimitOffsetPagination
+from rest_framework.response import Response
+
+
+class LimitOffsetPaginationExtended(LimitOffsetPagination):
```

Comment:
Not the most descriptive name, but I can't think of anything much better, so we can roll with it.